Change log level in eventlogmessage (#2336)
grafana/alloy: docs/sources/reference/components/loki/loki.process.md

When `drop_invalid_labels` is set to `true`, the stage drops fields that are not valid label names.
If set to `false`, the stage will automatically convert them into valid labels replacing invalid characters with underscores.

> Only lines that have the key:value format will be extracted. All non-alpha characters in the key are replaced with underscores.
> For example, `\tSecurity ID` will be available in the extracted map as `_Security_ID` to be used in later stages.

Reviewer: ```suggestion
The `eventlogmessage` stage only extracts lines with the key:value format.
All non-alpha characters in the key are replaced with underscores.
For example, `\tSecurity ID` is extracted as `_Security_ID`.
```

Could we change the wording to read like this?  I dropped the `to be used in later stages` because it felt out of place here. The focus is on the k:v pair extraction... not what is done later. Or have I misinterpreted the intent?
